apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -229,10 +232,10 @@ export default function ListBoxInline({ options = {} }) {
                 },
               }}
               selections={{
-                show: showToolbar,
+                show: true,
```

Comment:
we need to show selection toolbar all the time. The reason for this is more dropdown menu. 
if we render it conditionally ->  as soon as we click on any item from more menu -> since we dont show more menu after clicking it, 👇 happens: 
the ref for more menu gets removed from dom and both action will not happen at first, then we will lose the dropdown it self entirely. because the parent element that holds it's anchor is not rendering now! 

but if we render the action bar all the time, it will work like expected. 

https://user-images.githubusercontent.com/29652890/212701238-cad3aea7-5c0b-4bf8-96f5-bb334bb77c26.mov



### NOTE:
by doing this, we will not need `const [showToolbar, setShowToolbar] = useState(false);` state. removing this state entirely has impact on `useEffect` above at line 111, where we define `show`, `hide` functions. 

Any idea about what side effects we might expect from them?

Reply:
Unfortunately, this must be solved in another way since we cannot have the selections toolbar shown all the time. Also, note we are going to use this selections toolbar as it is for the new Filter pane.
Check the behaviour in other charts in sense-client and note that their selections toolbar is only shown after selections are made inside of the chart.

Reply:
In that case i would suggest to have a flag for showing selection toolbar in our options. If we set that variable when we embed then we should show it all the time for our use case. 

would that be a good idea you think?

Reply:
This on is now behind `options.shouldShowToolbar` flag. Also default value is `false`

Reply:
after talking with @johanlahti , we came into conclusion that we might involve @Caele and @T-Wizard into this. 
actually we have a demand in out designs to show our selection menu all the time in here: 
<img width="484" alt="Screenshot 2023-01-18 at 11 25 33" src="https://user-images.githubusercontent.com/29652890/213147474-30631752-203b-4067-8f90-f1a6caf5886f.png">

we've already got the confirmation that the look of it will not going to be like that but the fact of showing it all the time still standing as a silverpoint. 

So would having an options for showing it all the time would be a good idea?

Reply:
For the sake of the discussion, I'm just restating my thoughts about showing the selections toolbar all the time:
1. This would break with the behaviour of all other charts. In all other charts the selections toolbar is only shown once we perform selections.
2. The problem we see here seems to me like a short-coming (a bug or bad design) of this component, which could appear in other use-cases too.

But if the designers are fine with 1. (which I actually doubt) it could work as a temporary solution.